### PR TITLE
Update add-storage-space.md

### PR DESCRIPTION
**Pull request by msbemba**
updated the note to sort out the confusion. If the subscription was brought from VLSC or CSP the Add-on for extra storage can be purchased from them and not through microsoft.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2385

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/commerce/add-storage-space.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ description: "Learn to add and reduce file storage in your Microsoft 365 subscri
 If you start to run out of storage for your SharePoint Online site collections, you can add storage to your subscription if your plan is eligible. If you don't see the **Office 365 Extra File Storage** in the list of available add-ons, it means your plan is not eligible. For more information, see [Is my plan eligible?](#is-my-plan-eligible-for-office-365-extra-file-storage)
 
 > [!NOTE]
-> If you bought your subscription through Volume Licensing or a CSP, you can't buy **Office 365 Extra File Storage** for your organization. Contact your representative or partner for help.
+> If you bought your subscription through Volume Licensing or a CSP, you can't buy **Office 365 Extra File Storage** for your organization through MIcrosoft directly. Contact your representative or partner for help.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/add-storage-space.md]:
```suggestion
> If you bought your subscription through Volume Licensing or a CSP, you can't buy **Office 365 Extra File Storage** for your organization from Microsoft directly. Contact your representative or partner for help.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Review by cmcatee-MSFT — CHANGES_REQUESTED** [review]
Change to "directly from Microsoft"

**Comment by msbemba** [comment]
> Change to "directly from Microsoft"

I have made the changes @cmcatee-MSFT . Thank you.

**Pull request merged**